docs/config.md

```diff
@@ -72,6 +72,17 @@ All options can be overridden in `mango.local.yaml` (or `mango.local.json`) file
 
 ---
 
+### Images
+
+* `images` - array of all images - element can be:
+    * string (glob) of source image for minification or
+    * object for resize
+        * `src` - string (glob) source of images
+        * `sizes` - array of widths (int)
+        * `aspectRatio` - aspect ratio of image on output (float = width/height)
```

Comment:
Jak zachovat? Když bude jako string (jako to bylo doteď), tak se ho ani nedotkne